src/python/bin/aqueduct

```diff
@@ -214,6 +218,19 @@ def update_server_version():
         ]
     )
 
+    if parse_version(current_version) < parse_version("0.1") and parse_version(package_version) >= parse_version("0.1"):
```

Comment:
Probably fine for now, but we like need it to be a helper so that it's easier to add new tables in future.

Reply:
Yup when we have many of these, we can package each of them into a helper function called from this main function. I'll add a comment as a reminder.